test/e2e-tests.sh

```diff
@@ -14,7 +14,16 @@ failed=0
 
 (( !failed )) && install_service_mesh_operator || failed=2
 (( !failed )) && install_catalogsource || failed=3
-(( !failed )) && logger.success 'Cluster prepared for testing.' && run_e2e_tests || failed=4
+(( !failed )) && logger.success 'Cluster prepared for testing.'
+
+# Run serverless-operator specific tests.
+(( !failed )) && run_e2e_tests || failed=4
+
+# Setup serverless and run upstream e2e and conformance tests.
```

Comment:
Maybe we would like to have control, if that part should be run or not?

Reply:
Locally, maybe. The e2e-tests.sh script should be a uber script for CI imo though. We can always create smaller scripts that only run specific suites locally?